### ISSUE TITLE
How can i add metric logs in tensorboard in darts?

### ISSUE THREAD

**Issue by TemaDobryyR** [unit8co/darts#837]
I'm trying to figure out how to add metrics tracking to a tensorboard. I defined a tensorboard logger like that

`logger = TensorBoardLogger("darts_logs", 
                           name="nbeast",
                           log_graph=False, 
                           default_hp_metric=False)`

and passed this logger in pl_trainer_kwargs by dictionary. But i don't understand how i can track mape or rmse during training my model.
I read the pytorch documentation but still don't understand. i'l be very grateful for a sample code

**Comment by dennisbader** [comment]
You can set parameter `log_tensorboard=True` at model creation which will by autamatically create a logger that logs the train and validation loss to a `darts_logs` folder in your working directory.

For logging any additional metrics you would have to write a custom callback class and pass an instance of it at model creation like this: 

`pl_trainer_kwargs={"callbacks"=[MyCallbackClass()]}`.

See the PyTorch-Lightning documentation on custom callbacks: https://pytorch-lightning.readthedocs.io/en/stable/extensions/callbacks.html

Through the callback hooks you have access to the logger with `trainer.logger.experiment`.
